bin/setup

```diff
@@ -5,25 +5,29 @@ require 'pathname'
 APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
 
 Dir.chdir APP_ROOT do
-  # This script is a starting point to setup your application.
-  # Add necessary setup steps to this file:
-
   puts "== Installing dependencies =="
   system "gem install bundler --conservative"
   system "bundle check || bundle install"
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   system "cp config/database.yml.sample config/database.yml"
-  # end
+  puts "\n== Copying sample files =="
+  unless File.exist?("config/database.yml")
+    system "cp config/database.pg.yml config/database.yml"
+  end
+  unless File.exist?("certs/v2_key")
+    system "cp certs/v2_key.dev certs/v2_key"
+  end
+  unless File.exist?("REGION")
+    File.open("REGION", "w") { |f| f.puts "0" }
+  end
```

Comment:
Do you want it omitted? I included it just so I can find it to change, if/when I feel the need, without needing to look it up elsewhere.